tests/test_httperror.py

```diff
@@ -317,6 +317,57 @@ def _check(media_type, deserializer):
         # _check('application/x-yaml', yaml.load)
```

Comment:
TBH, I don't recall why this was commented out. It probably makes sense to uncomment it, and also add this check below.